lib/web/servers.go

```diff
@@ -80,3 +80,20 @@ func (h *Handler) getDesktopsHandle(w http.ResponseWriter, r *http.Request, p ht
 
 	return ui.MakeDesktops(windowsDesktops), nil
 }
+
+// getDesktopsHandle returns a desktop.
```

Comment:
```suggestion
// getDesktopHandle returns a desktop.
```

I would probably name this `handleGetDesktop` instead though.

Reply:
It seems as though we haven't been being particularly disciplined about this, but this aligns with @kimlisa 's suggested naming convention.

https://github.com/gravitational/teleport/blob/master/lib/web/apiserver.go#L393-L409